packages/core/src/pages/s.tsx

```diff
@@ -77,7 +77,7 @@ function generateSEOData(storeConfig: StoreConfig, searchTerm?: string) {
     : seo.description
 
   const canonical = searchTerm
-    ? `${storeConfig.storeUrl}/s?q=${searchTerm}`
+    ? `${storeConfig.storeUrl}/s?q=${searchTerm.replace(' ', '+')}`
```

Comment:
| Before | After |
|--------|--------|
| <img width="325" alt="Screenshot 2025-03-19 at 12 49 20" src="https://github.com/user-attachments/assets/c8d1746e-1dc9-4dd1-add3-55fc8bd921d1" /> | <img width="329" alt="Screenshot 2025-03-19 at 12 49 38" src="https://github.com/user-attachments/assets/f51c4610-a3e2-46e4-acb4-9fa377d7b8ac" /> | 
```suggestion
    ? `${storeConfig.storeUrl}/s?q=${searchTerm.replaceAll(' ', '+')}`
```

Reply:
agreed!

Otherwise won't be applied in this example `Handcrafted Cotton Salad`

<img width="686" alt="image" src="https://github.com/user-attachments/assets/f52d9411-c7ee-4887-bf96-19f32d40f35a" />